README.md

```diff
@@ -84,18 +84,22 @@ apt install libopenblas-base libomp-dev
 
 To install cuML from source, ensure the dependencies are met:
 
-
-1. [cuDF](https://github.com/rapidsai/cudf) (>=0.4.0)
+1. [cuDF](https://github.com/rapidsai/cudf) (>=0.5.0)
 2. zlib Provided by zlib1g-dev in Ubuntu 16.04
-3. cmake (>= 3.8, version 3.11.4 or 3.12.4 is recommended)
+3. cmake (>= 3.12, version 3.12.4 is recommended)
```

Comment:
I think we might just make it `cmake (>= 3.12.4)` since we know that 3.12.1-3 don't work (or at least not reliably due to a cuda cmake bug), and 3.13 up to 3.13.3 do.